docs/dashboards/custom-component.md

```diff
@@ -382,3 +382,57 @@ Dashboards.board('container', {
     }]
 });
 ```
+
+
+## Custom Component with data from the Connector
```

Comment:
In this whole text I believe Connector, data connector etc. should be replaced with `DataConnector`